app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,12 @@
+
+<% provide(:title, t(".About")) %>
```

Comment:
xóa line 1
tên biến ko bắt đầu với chữ cái viết hoa

dùng I18n cho toàn bộ text của các file html luôn á chú ơi

tag `a` thì thay bằng `link_to` của rails hì